tracer/transport.go

```diff
@@ -153,7 +183,10 @@ func (t *httpTransport) SendServices(services map[string]Service) (*http.Respons
 		return t.SendServices(services)
 	}
 
-	response.Body.Close()
+	if sc := response.StatusCode; sc != 200 {
+		return response, fmt.Errorf("SendServices expected response code 200, received %v", sc)
```

Comment:
I'm afraid we `spam` the user if we report all status code other than 200. Maybe @palazzem will know what behavior we want in this case?